Backend/schema/src/yaml/requests/GetMessageQueueRequest.yaml

```diff
@@ -0,0 +1,11 @@
+"$schema": "http://json-schema.org/draft-04/schema#"
+title: MessageQueue Request
+description: get queue of message request
+type: object
+additionalProperties: false
+properties:
+  messageQueueId:
+    description:  messageQueue Id is generated by system
+    type: string
+required:
+  - messageQueueId
```

Comment:
Should accountId also need to be added in the request? Should it have an authentication check?

Reply:
Good Catch!